src/elements/owncloud/Owncloud.wc.svelte

```diff
@@ -72,7 +72,7 @@
       symbol: "adminPassword",
       placeholder: "Admin Password",
       type: "password",
-      invalid: false,
+      validator: validateName, invalid: false
```

Comment:
i think you meant using `validatePassword` ?

![Screenshot from 2022-05-25 13-04-22](https://user-images.githubusercontent.com/32527347/170248420-0a2009b6-df03-46cb-a78f-cec2221d12ba.png)